app/views/restaurants/_comments.html.erb

```diff
@@ -0,0 +1,29 @@
+<!-- Comment area-->
```

Comment:
To add `comments` you should have a separate model, migration, association and controller for comments. I think you better start with just restaurants, then add this later when it's turn comes ;)